src/expression.c

```diff
@@ -9571,7 +9573,10 @@ Expression *DeleteExp::semantic(Scope *sc)
                 FuncDeclaration *fd = sd->dtor;
 
                 if (!f)
+                {
+                    semanticTypeInfo(sc, ts);
```

Comment:
Unfortunately it's impossible. Currently a struct TypeInfo may refer to template member functions (`opEquals` and `opCmp`), and their mangled name is affected by the attribute inference result. So we should guarantee to run those semantic3 pass before the reach to glue layer. That's why the `semanticTypeInfo` call is necessary in each places.